### DISCUSSION TITLE
removing unwanted pools made in self pooling

### DISCUSSION THREAD

**Discussion by pericpera** [Chia-Network/chia-blockchain#7382] [Support]
Dear,
How to remove pool made by self pooling that does not want, there are no plots for it. It is CHIA GUI under w10. There is no remove or delete option from the list. It is an automatically generated pool name.
Best regards

**Comment by MaximusChia** [comment]
The same, thats why i just created a single NFT, Chia client neesd many things to improve and polish

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
